OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -140,6 +143,20 @@ Order IIssueDeployOrder.IssueDeployOrder(Actor self, bool queued)
 
 		bool IIssueDeployOrder.CanIssueDeployOrder(Actor self) { return !IsTraitPaused && !IsTraitDisabled; }
 
+		bool IsGroupDeployNeeded(Actor self)
+		{
+			var actors = self.World.Selection.Actors.ToList();
+
+			foreach (Actor actor in actors)
```

Comment:
`var a(ctor) in actors`

Reply:
The main point here was that the code convention is to use implicit types where possible. The `var` bit still stands.

Reply:
Ohhh right! My bad. Fixed in the latest commit!